gapis/resolve/framebuffer_attachment.go

```diff
@@ -68,6 +68,10 @@ func typeToString(t api.FramebufferAttachmentType) string {
 		return "Color"
 	case api.FramebufferAttachmentType_OutputDepth:
 		return "Depth"
```

Comment:
nit: do we want to change these two to `OutputColor` and `OutputDepth` as well?

Reply:
I rarely hear people use the qualification of "output" for these attachments like they do with input attachments. I am open to changing this.